crates/services/p2p/src/request_response/messages.rs

```diff
@@ -49,6 +50,7 @@ pub enum RequestMessage {
 #[derive(Serialize, Deserialize, Debug, Clone)]
 pub enum ResponseMessage {
     SealedBlock(Option<SealedBlock>),
+    SealedBlocksInclusive(Vec<SealedBlock>),
```

Comment:
```suggestion
    SealedBlocksInclusive(Vec<Option<SealedBlock>>),
```

Maybe it is better to follow the pattern=)